project-reports/2024/2024-annual-Hyperledger-Besu.md

```diff
@@ -0,0 +1,43 @@
+---
+layout: default
+title: 2024 Annual Review Hyperledger Besu
+parent: 2024
+grand_parent: Project Updates
+---
+
+# Project Health
+[LFX Insights for Besu](https://insights.lfx.linuxfoundation.org/foundation/hyp/overview?project=besu&bestPractice=false). 
+The Hyperledger Besu project is quite healthy in terms of overall contributions. We have seen our unique contributor numbers creeping up over the last yea and increase by over 200 unique contributors. Active contributor numbers have fluctuated, but the project has added new maintainers and new organizations of note, like Citi and Kaleido. Consensys Software makes up around 87% of the maintainer base, but is committed to the project. Other organizations make up 13% of maintainers/contributors and we hope to see this increase. Overall, the project maintains consistent releases, has multiple committed organizations, and is becoming more and more relevant in Ethereum and Ethereum-like infrastructure. We also saw the founding of the Hyperledger Besu for Financial Services Working Group to further localized goals around finance use-cases with the technology. 
+
+# Maintainer Diversity
+
+[Maintainers File.](https://github.com/hyperledger/besu/blob/main/MAINTAINERS.md) Consensys Software makes up around 87% of the maintainer base, but is committed to the project. Maintainer diversity increased this year by adding Kaleido as a contributing organization, alongside contributions from Web3Labs. Swirlds Labs also has new contributors moving toward maintainer status in 2024. We are committed to increasing the diversity on this list by encouraging more development on the private network code-base by those using the technology. Existing maintainers have given special attention to new pull requests and are very timely in moving those forward. 
+
+
+# Project Adoption	
+
+The Besu project has been adopted by Linea, a layer 2 network developed by Consensys. It is now powering the sequencer that is used to produce blocks and advance the network as critical infrastructure. Citi Bank has also joined the Hyperledger Foundation as a member organization using Hyperledger Besu. Several new institutional staking providers have adopted Besu in their tech stacks to support Ethereum client diversity on Mainnet. Besu sits in 3rd place alongside four other execution clients. It is also being forked for use in other layer 2 networks. 
```

Comment:
on the permissioned deployment side, Besu's adoption can be seen in many project. as a public source the HL foundation's own CBDC eBook listed Besu for a number of countries' CBDC projects: https://www.linuxfoundation.org/hubfs/Hyperledger_CBDC%20ebook_V2.pdf?hsLang=en. would be good to add that to show it's adoption is not limited to public chains.